bin/dotd

```diff
@@ -236,6 +237,35 @@ def print_dtp_message
   end
 end
 
+# Retrieves recent Honeybadger issues, displaying them on the console.
+def get_honeybadger_issues
+  issues = Honeybadger.get_recent_issues
+  issues.sort_by! do |issue|
+    [issue[:project], issue[:environment], (issue[:assignee] ? issue[:assignee] : 'ZZZ')]
+  end
+
+  puts '' if issues.count > 0
+  issues.each do |issue|
+    environment = issue[:environment] == 'production' ? (red issue[:environment]).to_s : (bold issue[:environment]).to_s
+    project = bold issue[:project]
+    message = <<-EOS.unindent
+      [#{project}, #{environment}]: #{issue[:message]}
+          #{issue[:url]}
+    EOS
+    # TODO(asher): Eliminate this hackiness, for some reason, some extra newlines are showing up at
+    # the end of the message. This seems correlated with environment and project.
+    message = message.strip
+    if issue[:assignee]
+      puts dim message
```

Comment:
Note: Because of how terminal colors work, you're only dimming the `[project` part of your string here - the color applied to the `environment` bit takes over after that.  You might instead want to extract out `issue[:message]` and conditionally dim that before assembling everything, instead.